lib/Sema/DerivedConformanceDifferentiable.cpp

```diff
@@ -346,7 +346,6 @@ static ValueDecl *deriveDifferentiable_method(
   funcDecl->setGenericSignature(parentDC->getGenericSignatureOfContext());
   funcDecl->computeType();
   funcDecl->copyFormalAccessFrom(nominal, /*sourceIsParentContext*/ true);
-  funcDecl->setValidationToChecked();
```

Comment:
Should we use `DeclValidationRAII` here and the remaining places? Tests pass without it.

Reply:
If it's obvious how to use `DeclValidationRAII`, a patch would be appreciated! Otherwise, I think deferring is fine.

Reply:
It is obvious how to use it, but couldn't tell when to use it after a cursory inspection of existing code - some places use it, some don't. I will defer for now.